otter/log/formatters.py

```diff
@@ -332,11 +244,8 @@ def Observer(eventDict):
             if key not in PRIMITIVE_FIELDS:
                 log_params[key] = value
 
-        # emit an audit log entry also, if it's an audit log
         if 'audit_log' in eventDict:
```

Comment:
Although hmm... maybe I can figure out what the specifically elasticsearch query is to list all dashboards as JSON, and search that way.  Kibana 3 limits the number displayed, doesn't actually create a DOM element for every one and hide them - it just creates a set number and hides/repopulates them.

Although there is still that one audit log dashboard.